x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/utils.ts

```diff
@@ -53,24 +54,45 @@ export interface BulkError {
 
 export const createBulkErrorObject = ({
   ruleId,
+  id,
   statusCode,
   message,
 }: {
-  ruleId: string;
+  ruleId?: string;
+  id?: string;
   statusCode: number;
   message: string;
 }): BulkError => {
-  return {
-    rule_id: ruleId,
-    error: {
-      status_code: statusCode,
-      message,
-    },
-  };
+  if (id != null) {
```

Comment:
I am wondering if we should be aware of  `ruleId` and `id`, if both are presents. it might be interesting for pre-packaged rules.

Reply:
my two cents 💸

Reply:
Those are typically mutually exclusive and I think 100% at the moment. If not, we would end up bubbling the two up more but the advice for the UI is still if you have a valid `id` use that over a `rule_id` to minimize issues if we end up with duplication bugs.

Reply:
I do understand that, but my feeling was more about the pre-packaged rules where people will be more aware of the rule_id than the id. So it might be useful for the user to know the rule_id and id if something went wrong. If I did understand this correctly is to bring the id if there is an error.

Reply:
In those cases you can call the API with just the `rule_id` and it will only output the error with the rule_id. So it listens to what things above tell it and I think the things above it are coded correctly?

Reply:
I added the feature and fixed the tests for you 👍so you can call it either way and if you give it both id and rule_id it will return both for you.